include/integratorxx/atomic_densities/SlaterAtomicShell.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+#include <vector>
+#include <cmath>
+
+class SlaterTypeAtomicShell {
+  /// Angular momentum
+  int angular_momentum;
+  /// Exponents
+  std::vector<double> exponents;
```

Comment:
I would `typedef` these containers to make the code a bit more readable. e.g.
```
using int_container = std::vector<int>;
using real_container = std::vector<double>;
using coefficient_container = std::vector<real_container>;
```

Reply:
done